ROPC warnings (#4849)
dotnet/EntityFramework.Docs: entity-framework/core/dbcontext-configuration/index.md

                options => options.UseSqlServer("name=ConnectionStrings:DefaultConnection"));
        }
-->

[!code-csharp[snippet_1](../../../samples/core/Miscellaneous/ConfiguringDbContext/WebApp/Program9.cs?name=snippet_1)]

> The preceding code registers `ApplicationDbContext`, a subclass of `DbContext` as a scoped service in the ASP.NET Core app service provider. The service provider is also know as the dependency injection container. The context is configured to use the SQL Server database provider and reads the connection string from ASP.NET Core configuration.

Reviewer: ```suggestion
The preceding code registers `ApplicationDbContext`, a subclass of `DbContext`, as a scoped service in the ASP.NET Core app service provider. The service provider is also known as the dependency injection container. The context is configured to use the SQL Server database provider and reads the connection string from ASP.NET Core configuration.
```

Reviewer 2: ```suggestion
The preceding code registers `ApplicationDbContext`, a subclass of `DbContext`, as a scoped service in the ASP.NET Core app service provider. The service provider is also known as the dependency injection container. The context is configured to use the SQL Server database provider and [read the connection string from ASP.NET Core configuration](xref:core/miscellaneous/connection-strings#aspnet-core).
```

-->
[!code-csharp[UseNewForWebApp](../../../samples/core/Miscellaneous/ConfiguringDbContext/WebApp/UseNewForWebApp.cs?name=UseNewForWebApp)]

> ## Using a DbContext factory, e.g. for Blazor

Reviewer: ```suggestion
## Use a DbContext factory
```
